test/jdk/TEST.ROOT

```diff
@@ -9,17 +9,41 @@
 # should be taken to handle test failures of intermittent or
 # randomness tests.
 #
-# A "headful" test requires a graphical environment to meaningfully
-# run. Tests that are not headful are "headless".
-# A test flagged with key sound needs audio devices on the system, this
-# may be accompanied by the headful keyword since audio device access
+# A test flagged with cgroups uses cgroups.
+#
+# Notes on "client" keywords : headful sound printer multimon 
+# ===========================================================
+#
+# These keywords are there to help with test selection so that
+# tests that need a particular resource can be selected to run on a system
+# with that resource. Conversely "!somekeyword" can be used to exclude tests
```

Comment:
probably we can link this from our wiki page where we describe the usage of keywords?:
https://wiki.openjdk.org/display/ClientLibs/Automated+client+GUI+testing+system+set+up+requirements

BTW the "doc" has nothing related to printers right now as well:
https://github.com/openjdk/jdk/blob/master/doc/testing.md

Reply:
> probably we can link this from our wiki page where we describe the usage of keywords?: https://wiki.openjdk.org/display/ClientLibs/Automated+client+GUI+testing+system+set+up+requirements


Good idea, once this is pushed, I'll add that right where we discuss the keywords. If I remember !

> 
> BTW the "doc" has nothing related to printers right now as well: https://github.com/openjdk/jdk/blob/master/doc/testing.md

Perhaps this is a bit too specialised to discuss there.